commands/client-list.md

```diff
@@ -40,7 +40,8 @@ b: the client is waiting in a blocking operation
 i: the client is waiting for a VM I/O (deprecated)
 d: a watched keys has been modified - EXEC will fail
 c: connection to be closed after writing entire reply
-u: the client is unblocked
+U: the client is unblocked
```

Comment:
`u` = `This client was unblocked`

`U` = `Client connected via Unix domain socket`